storagenode/storagenodedb/infodb.go

```diff
@@ -221,7 +221,7 @@ func (db *InfoDB) Migration() *migrate.Migration {
 				Description: "Add creation date.",
 				Version:     6,
 				Action: migrate.SQL{
-					`ALTER TABLE pieceinfo ADD COLUMN piece_creation TIMESTAMP`,
+					`ALTER TABLE pieceinfo ADD COLUMN piece_creation TIMESTAMP NOT NULL DEFAULT '1970-01-01 00:00:01'`,
```

Comment:
We define similar columns on the satellite as `TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT 'epoch'`

Perhaps, do the same here for consistency.

Reply:
I think that the `WITH TIME ZONE` declaration now fails the tests. Maybe it was not a good idea for SQLite.